en/docs/monitoring-and-insights/alerts-overview.md

```diff
@@ -0,0 +1,145 @@
+# Alert Overview
+
+This section explains how you can configure alerts for your Choreo components. Setting up alerts allows you to proactively monitor your components ecosystem and take corrective measures when necessary.
+
+You can configure alerts for each environment within your organization. You can add, modify, or delete alerts per component. Optionally, you can specify a list of emails for each alert configuration.
+
+!!! info
+    - You can configure a maximum of 10 alerts per component.
+    - You can add a maximum of 5 email addresses per alert.
+
+Alerts can be categorized as [latency alerts](#latency-alerts) , [traffic alerts](#traffic-alerts) , [resources alerts](#resources-alerts) , [logs alerts](#logs-alerts), [build failure alerts](#build-failure-alerts) and [status code alerts](#status-code-alerts).
+
+## Supported Alert Types
+
+### Latency alerts
+
+Latency alerts notify you if the response latency of a component exceeds a predefined threshold in a given time period. This is useful for components that need to meet specific SLAs and for proactively identifying slow components.
+
+Configurable Parameters:
+
+- Metric: 99th, 95th, 90th or 50th percentile.
+- Threshold: Latency in milliseconds (e.g., 1800 ).
+- Period: Duration the threshold must be exceeded (e.g., 5 minutes).
+
+### Traffic alerts
+
+Traffic alerts notify you when the request count of a component exceeds a predefined threshold. This is useful for managing components with backend traffic limits or monetized backends that require proactive scaling based on incoming traffic.
+
+Configurable Parameters:
+
+- Threshold: Requests per minute (e.g., 200).
+- Period: Monitoring window (e.g., 5 minutes).
+
+### Resources alerts
+
+Resource alerts notify you when your component’s CPU or memory usage exceeds the defined thresholds. This ensures you can fix the resources allocations early to avoid performance issues or downtimes.
+
+Configurable Parameters:
+
+- Metric: CPU or Memory.
+- Threshold: **mCPU** for CPU and **MiB** for Memory(e.g., 1000).
+- Period: Duration the threshold must be exceeded (e.g., 5 minutes).
+
+    !!! Tip
+        - **CPU**: mCPU (milliCPU) measures CPU usage in fractions of a core, where 1000m = 1 full core.
+        - **Memory**: MiB (Mebibyte) measures memory in binary units, where 1 MiB = 2^20^ bytes.
+
+### Logs alerts
+
+Logs alerts trigger notifications when a specific phrase appears frequently in your component logs. This helps to identify recurring issues or critical errors quickly, enabling faster troubleshooting.
```

Comment:
Can't say recurring because users can configure to get alerts even for 1 occurrence